solutions/bronze/usaco-965.mdx

```diff
@@ -169,6 +169,57 @@ public class Lineup {
 **Time Complexity:** $\mathcal{O}(N)$<Asterisk>Note that we don't have to include the time it takes to build permutations and iterate through them because they're constant factors.</Asterisk>
 
 <LanguageSection>
+<PySection>
+
+```py
```

Comment:
1. move this below the cpp sol
2. type annotations
3. use docstrings instead of comments in the functions